### PR TITLE
Bugfix - Typos

### PR DESCRIPTION
**Pull request by jutaz**
Found few typos in `./Documentation` folder, so I thought I’m gonna fix 'em, and here we are.

### CODE REVIEW

**Comment by KamranMackey** [comment]
You need to submit this as a patch to the Linux Kernel Mailing List, as Linus doesn't look at PRs.

**Comment by jobs-git** [comment]
Some are not really typo errors like: `behaviour` in british english and `behaviour` in american english. British guy checks your work, then corrects your `typo`, then there come another American guy who see the British guy's `typo`, then the cycle of `typo errors` continues.

**Comment by akkatracker** [comment]
Most of these are not typos as they are in British English as @jobs-git said.

**Comment by ancaemanuel** [comment]
Tutorial: https://www.youtube.com/watch?v=LLBrBBImJt4
Setting git send mail: https://coderwall.com/p/dp-gka/setting-up-git-send-email-with-gmail
Use get_maintainer.pl
Use the mailing lists.
Close this please, and tell to others.

File: Documentation/edac.txt [hunk 1]
```diff
@@ -658,7 +658,7 @@ exports one
 		echo 2 >/sys/devices/system/edac/mc/mc0/inject_addrmatch/dimm
 		echo 1 >/sys/devices/system/edac/mc/mc0/inject_addrmatch/rank
 
-	To return to the default behaviour of matching any, you can do:
+	To return to the default behavior of matching any, you can do:
 		echo any >/sys/devices/system/edac/mc/mc0/inject_addrmatch/dimm
```
Comment (macaris64) [inline Documentation/edac.txt]: [](url)

**Comment by rahulxcode** [comment]
how to pull a request ?

**Pull request closed without merging**